src/model.py

```diff
@@ -151,6 +153,14 @@ def init_engine(self):
             AsyncEngineArgs(**self.vllm_engine_config)
         )
 
+        # Create vLLM custom Metrics
+        labels = {
+            "model": self.args["model_name"],
+            "version": self.args["model_version"],
+        }
+        logger = VllmStatLogger(labels=labels)
+        self.llm_engine.add_logger("triton", logger)
+
```

Comment:
Question for this and `gauge/counter` PR as well. Would this cause trouble if `tritonserver` was built without metrics ?

Reply:
@rmccorm4 how will the backend APIs behave ^^

Reply:
My concern comes from here: https://github.com/triton-inference-server/core/pull/386/files#diff-5ed8dbbcfda4302eb8115993e5297cdf1e184f856c415059946525a7b618397cR3461-R3464
If metrics are not enabled, we send an error. IIUC this and `gauge` PR will make vllm backend use metrics all the time?

Reply:
Please use this [discussion](https://github.com/triton-inference-server/vllm_backend/pull/53#discussion_r1697679955) to track the issue.